web/src/lib/components/admin-page/jobs/jobs-panel.svelte

```diff
@@ -61,12 +61,12 @@
     [JobName.ThumbnailGeneration]: {
       icon: mdiFileJpgBox,
       title: getJobName(JobName.ThumbnailGeneration),
-      subtitle: $t('thumbnail_generation_job_description'),
+      subtitle: $t('admin.thumbnail_generation_job_description'),
     },
     [JobName.MetadataExtraction]: {
       icon: mdiTable,
       title: getJobName(JobName.MetadataExtraction),
-      subtitle: $t('metadata_extraction_job_description'),
+      subtitle: $t('admin.metadata_extraction_job_description'),
     },
     [JobName.Library]: {
```

Comment:
Does this jobname not require admin.? Should these not all be prefixed with that? @danieldietzler ?